x-pack/plugins/ml/public/file_datavisualizer/components/edit_flyout/overrides.test.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+import { shallow } from 'enzyme';
+import React from 'react';
+
+import { Overrides } from './overrides';
+
+describe('Overrides', () => {
+
+  test('render overrides', () => {
+    const props = {
+      setOverrides: () => {},
+      overrides: {},
+      defaultSettings: {},
+      setApplyOverrides: () => {},
+    };
+
+    const component = shallow(
+      <Overrides {...props} />
+    );
+
+    expect(component).toMatchSnapshot();
+  });
+});
```

Comment:
Great to have the full snapshot test here! It'd be a good idea to add tests for the conditionals for the form rows in the render as well. Just checking that a particular label with the expected text exists when a condition is met (or something like that) should be enough.
This way we can make sure what's expected is showing up under different conditions and it's not as susceptible to implementation detail changes as snapshots.

Reply:
yes, i agree. This test was added as a placeholder with the intention of adding tests to cover everything at a later stage.
The rest of the tests will be added in a subsequent PR.